.github/workflows/redmine-upkeep.yml

```diff
@@ -87,6 +84,7 @@ jobs:
              --git-dir=./ceph/
         env:
           REDMINE_API_KEY: ${{ secrets.REDMINE_API_KEY_BACKPORT_BOT }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think you can expose it at workflow level. Do you intentionally nest it?

Reply:
No but probably it's better to be selective which commands we share secrets with anyway.